src/helpers/formatting.ts

```diff
@@ -0,0 +1,31 @@
+const isPositive = (value: string) => {
+  return Number(value) > 0;
+};
+
+export const toDate = (textDate: string) => {
+  const today = new Date();
+
+  let dd = textDate.split('/')[0];
```

Comment:
não daria pra usar [Destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) aqui? mais ou menos assim:

```suggestion
  let [dd, mm, yyyy] = textDate.split('/');
```